.github/actions/package-nfpm/action.yml

```diff
@@ -60,19 +60,15 @@ runs:
         else
           export DIST=""
         fi
-
-        luaver=$(lua -e "print(string.sub(_VERSION, 5))" 2>/dev/null || echo 0)
-        echo "luaver is $luaver"
-        if [ $luaver = "0" ]; then
-          echo "Cannot get lua version"
-          exit 1
-        fi
```

Comment:
if you remove this section, this will break centreon-stream-connectors libs, isn't it ?